### PR TITLE
tlacuache-thesis-fc-unam:0.1.1

### PR DESCRIPTION
**Pull request by davidalencia**
I am submitting
- [x] a new package
- [ ] an update for a package

This is a thesis template for the Science Faculty at Universidad Nacional Autónoma de México (UNAM) based on my thesis.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/fc-thesis/0.1.0/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Jonas Neugebauer
```
Comment (laurmaedje) [inline packages/preview/fc-thesis/0.1.0/LICENSE:3]: This looks like a copy paste error

**Review by laurmaedje — COMMENTED** [review]
The name is a bit too descriptive as-is. I assume fc stands for "facultad de ciencias" which is probably the official abbreviation of the faculty at your institution? I recommend to include some unique, creative component and the term `unam` in the name. Also see the [naming guidlines in the README](https://github.com/typst/packages?tab=readme-ov-file#submission-guidelines) and the [list of other templates](https://typst.app/universe/search?kind=templates) for inspiration.

**Comment by davidalencia** [comment]
Ok, I understand, to make the name less obvious I added the name of a random Mexican animal.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**